src/tests/fuzz/fuzz_applayerparserparse.c

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file
+ * @author Philippe Antoine <contact@catenacyber.fr>
+ * fuzz harness for AppLayerProtoDetectGetProto
+ */
+
+
+#include "suricata-common.h"
+#include "app-layer-detect-proto.h"
+#include "flow-util.h"
+#include "app-layer-parser.h"
+
+#define HEADER_LEN 6
+
+void fuzz_openFile(const char * name) {
+}
+
+AppLayerParserThreadCtx *alp_tctx = NULL;
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
```

Comment:
style: variable & argument names should be all lowercase

Reply:
Ok, I was using libfuzzer style https://llvm.org/docs/LibFuzzer.html#id22